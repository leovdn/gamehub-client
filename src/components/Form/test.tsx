import { FormLink, FormWrapper } from './index'
import { renderWithTheme } from 'utils/tests/helpers'

describe('<Form />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(
      <FormWrapper>
        <FormLink>
          From Link Test<a href="#">Form Link</a>
        </FormLink>
      </FormWrapper>
    )

    expect(container.parentElement).toMatchSnapshot()
  })
})
