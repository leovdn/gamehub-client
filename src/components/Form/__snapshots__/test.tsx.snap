// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> should render the heading 1`] = `
.c0 {
  text-align: center;
  font-size: 1.4rem;
}

.c0 a {
  color: #3CD3C1;
  text-decoration: none;
  border-bottom: 0.1rem solid #3CD3C1;
  transition: color,border 0.1s ease-in-out;
}

.c0 a:hover {
  border-bottom: 0.1rem solid #29b3a3;
  color: #29b3a3;
}

<body>
  <div>
    <main
      class=""
    >
      <span
        class="c0"
      >
        From Link Test
        <a
          href="#"
        >
          Form Link
        </a>
      </span>
    </main>
  </div>
</body>
`;
